src/main/java/pl/coderslab/web/Login.java

```diff
@@ -1,15 +1,17 @@
-package pl.coderslab.model;
+package pl.coderslab.web;
+
+import lombok.extern.java.Log;
```

Comment:
powinniśmy się zdecydować, którego loggera chcemy używać.

Reply:
fakt, dotychczas używaliśmy loggera z dependencji - to usunę tego loggera z tego pliku.